### ISSUE TITLE
lbwps_enabled not working

### ISSUE THREAD

**Issue by zszymczyk** [arnowelzel/lightbox-photoswipe#3]
Hi I was trying to use the function:

function my_lbwps_enabled($id, $enabled)
{
	if( function_exists( 'is_product' ) )
	{
		if( is_front_page() ) return false;
	}

	return $enabled;
}

add_filter( 'lbwps_enabled', 'my_lbwps_enabled', 10, 2 );

but it doesn't work for me. I know that I can exclude the front page in admin but I need to exclude it also from many other pages (all pages from custom types).

**Comment by arnowelzel** [comment]
What exactly do you want to achieve?

The function is an **example** how to disable PhotoSwipe on **product pages of WooCommerce** (that's clearly indicated in the comment above the code: `// Disable Lightbox with PhotoSwipe on WooCommerce product pages`). If you want to use that for custom types you have of course to create your own function.

This following as also an **example** how to suppress PhotoSwipe for any type except "page" and "post". **You must modify this for your own needs:**

```
function my_lbwps_enabled($id, $enabled)
{
    // get current post type
    $post_type = get_post_type();

    // if this not "post" or "page" then don't use PhotoSwipe
    if ($post_type != 'post' && $post_type != 'page') {
        return false;
    }
    
    // Default: let PhotoSwipe decide itself
    return $enabled;
}

add_filter('lbwps_enabled', 'my_lbwps_enabled', 10, 2);
```

Further reading:

https://developer.wordpress.org/reference/functions/get_post_type/

**Comment by zszymczyk** [comment]
Hi,
thank you.

Yes, I understand the example with product pages of WooCommerce. And I did exactly the same for post_type as you in the above example. But it doesn't work. All .js files all still there (clear cache). The only way that work is to add id in admin panel but i have to many pages to exclude. 

I have this in my functions.php file:

/*
 * Lightbox for Photoswipe
 */
function my_lbwps_enabled($id, $enabled)
{
    // get current post type
    $post_type = get_post_type();

    // if this not "post" don't use PhotoSwipe
    if ($post_type != 'post') {
        return false;
    }
    
    // Default: let PhotoSwipe decide itself
    return $enabled;
}

add_filter('lbwps_enabled', 'my_lbwps_enabled', 10, 2);

I looks like the photoswipe doesn't work but all .js files are loaded.

**Comment by arnowelzel** [comment]
I see - yes, this is a bug, even though it works the scripts should not be enqueued. I fixed this with a4e693d74edf9f11040504df68858e96af5a0347.

**Comment by zszymczyk** [comment]
Hi, thank you for your time and fix. Unfortunately I think that not everything is fixed yet.

In the source code I still see this:
`<div class="pswp" tabindex="-1" role="dialog" aria-hidden="true">
    <div class="pswp__bg"></div>
    <div class="pswp__scroll-wrap">
        <div class="pswp__container">
            <div class="pswp__item"></div>
            <div class="pswp__item"></div>
            <div class="pswp__item"></div>
        </div>
        <div class="pswp__ui pswp__ui--hidden">
            <div class="pswp__top-bar">
                <div class="pswp__counter"></div>
                <button class="pswp__button pswp__button--close" title="Close (Esc)"></button>
                <button class="pswp__button pswp__button--share" title="Share"></button>
                <button class="pswp__button pswp__button--fs" title="Toggle fullscreen"></button>
                <button class="pswp__button pswp__button--zoom" title="Zoom in/out"></button>
                <div class="pswp__preloader">
                    <div class="pswp__preloader__icn">
                      <div class="pswp__preloader__cut">
                        <div class="pswp__preloader__donut"></div>
                      </div>
                    </div>
                </div>
            </div>
            <div class="pswp__share-modal pswp__share-modal--hidden pswp__single-tap">
                <div class="pswp__share-tooltip">
                </div> 
            </div>
            <button class="pswp__button pswp__button--arrow--left" title="Previous (arrow left)"></button>
            <button class="pswp__button pswp__button--arrow--right" title="Next (arrow right)"></button>
            <div class="pswp__caption">
                <div class="pswp__caption__center"></div>
            </div>
        </div>
    </div>
</div>`

Unfortunately it breaks the page because at the far bottom I see small buttons generated by above code.

**Comment by arnowelzel** [comment]
Fixed with 4d2dde98e722be6d413d0355926aeacc5a0f8e24
